### PR TITLE
Bulk pep8 fixes - hand crafted

### PR DESCRIPTION
**Pull request by gundalow**
##### SUMMARY
Fix by hand the remaining issues that autopep8 couldn't

`autopep8` fixes were done in https://github.com/ansible/ansible/pull/33682

##### ISSUE TYPE
 - Bugfix Pull Request

### CODE REVIEW

**Comment by gundalow** [comment]
bot_skip

**Comment by gundalow** [comment]
deleted

**Comment by sivel** [comment]
@gundalow 

##### W503 line break before binary operator:

Instead of:

```
            elif (awaited_state == 'InService'
                  and instance_state.reason_code == "Instance"
                  and time.time() >= wait_timeout):
```

It should be:

```
            elif (awaited_state == 'InService' and
                  instance_state.reason_code == "Instance" and
                  time.time() >= wait_timeout):
```

##### E265 block comment should start with '# '

```
#----------------------------------------------------
```

Should be

```
# ---------------------------------------------------
```

Basically, there needs to be a space after `#`

**Comment by gundalow** [comment]
Yup, worked out W503

E265 Ah, thanks. I wasn't sure if there was a nicer way of doing the larger comment blocks

File: lib/ansible/modules/cloud/amazon/ec2_elb.py [hunk 1]
```diff
@@ -212,9 +212,9 @@ def _await_elb_instance_state(self, lb, awaited_state, initial_state, timeout):
             elif self._is_instance_state_pending(instance_state):
                 # If it's pending, we'll skip further checks and continue waiting
                 pass
-            elif (awaited_state == 'InService'
-                  and instance_state.reason_code == "Instance"
-                  and time.time() >= wait_timeout):
+            elif (awaited_state == 'InService' and
+                  instance_state.reason_code == "Instance" and
+                  time.time() >= wait_timeout):
```
Comment (abadger) [inline lib/ansible/modules/cloud/amazon/ec2_elb.py]: PEP8 specifies the opposite.  Break before binary operators: https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator

**Comment by abadger** [comment]
the binary operator changes are bad.  We should revert those.  Looking at the version difference, it looks like pycodestyle has decided that for the current major version they are just going to ignore W503.  So I suggets we do the same: https://github.com/PyCQA/pycodestyle/issues/498

It looks like they are still undecide as to whether they'll implement the opposite check (break before binary operators) when they do a new major version.

**Comment by abadger** [comment]
Other than the binary operator breaking, this looks good.  +1 to merge once that's corrected.

**Comment by gundalow** [comment]
One everyeone is happy Ansibot needs automerge enabling again https://github.com/ansible/ansible/commit/87776d75dc39847c5223a3268063bc3d4eb30235#diff-5baefb0f9c8093094911cec657444e70

**Comment by abadger** [comment]
+1 from me.  @sivel, Is this good for you too?

**Comment by abadger** [comment]
I've gone ahead and merged this.  We can always revisit if we want to use a different strategy for W503

**Comment by sivel** [comment]
Apologies for the late response, +1

**Pull request merged**